<launch>
  <node pkg="dhand_control"  type="dhand_joint_state_republisher"  name="dhand_joint_state_republisher" output="screen">
    <param name="joint_state_republish_topic_name" value="/joint_states" />
    <param name="joint_state_subscribe_topic_name" value="/sia5/joint_states" />
  </node>

</launch>
